.github/workflows/ghpages-deploy.yml

```diff
@@ -1,37 +1,35 @@
 name: deploy-docusaurus
 
 on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+    push:
+        branches: [master]
 
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  publish:
-    runs-on: ubuntu-latest
-    steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - name: Check out repo
-        uses: actions/checkout@v2
-      # Node is required for npm
-      - name: Set up Node
-        uses: actions/setup-node@v2
-        with:
-          node-version: "14"
-      # Install and build Docusaurus website
-      - name: Build Docusaurus website
-        run: |
-          npm install 
-          npm run build
-      - name: Deploy to GitHub Pages
-        if: success()
-        uses: crazy-max/ghaction-github-pages@v2
-        with:
-          target_branch: gh-pages
-          build_dir: build
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+            - name: Check out repo
+              uses: actions/checkout@v2
+            # Node is required for npm
+            - name: Set up Node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "14"
```

Comment:
why such an old version?

Reply:
14 is listed as the minimum requirement, there really isn't any reason to use any particular version

Reply:
I've set it to 16 now since that's LTS. That's probably the safest bet